frontApp/app/profile.tsx

```diff
@@ -0,0 +1,221 @@
+                                <Ionicons name="checkmark-circle-sharp" size={32} color="black" />
+                            </View>
+                            <TextInput style={styles.linkText} value = "Verify my Phone" editable = {false}/> 
+                        </View>
+                </View>
+                <View style={styles.edtBtnTitleText} >
+                    <TouchableOpacity style={styles.edtBtn}>
+                        <View style={styles.profileView}>
+                            <Text style={styles.edtBtnText}>Edit Profile</Text>
+                            <Ionicons name="create" size={32} color="#808080" />
+                        </View>
+                    </TouchableOpacity>
+                </View>
+            </ScrollView>
+        </View>
+    );
+};
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        backgroundColor: "#e0a5a5",
+        marginBottom: '0%',
+    },
+    header: {
+        flexDirection:"row",
+        marginBottom: '1%',
+        marginTop: '15%',
+        marginLeft: '33%'
+    },
+
+    header2: {
+        flex: 3
+    },
+    headerText: {
+        color: '#000000',
+        marginLeft: '6.5%',
+        fontSize: 36,   
+        fontWeight: 'bold',
+        justifyContent: 'flex-end'
+    },
+    profileGenTitleText: {
+        color: '#000000',
+        fontWeight: 'bold',
+        width: '66%',
+        textAlign: 'left',
+        marginTop: '10%',
+        marginLeft: '11%',
+        marginBottom: '1%',
+        fontSize: 19,
+    },
+    profileVerTitleText: {
+        color: '#000000',
+        fontWeight: 'bold',
+        width: '66%',
+        textAlign: 'left',
+        marginTop: '2%',
+        marginLeft: '11%',
+        marginBottom: '3%',
+        fontSize: 19, 
+    },
+    profileView: {
+        flexDirection: 'row',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+    },
+    accountText: {
+        width: '70%',
+        borderRadius: 15,
+        height: 25,
+        alignItems: 'center',
+        marginLeft: '2%',
+        marginBottom: '3%',
+        marginTop: '4%',
+        justifyContent: 'center'
+    },
+    linkText:{
+        color: '#0000EE',
+        width: '70%',
+        borderRadius: 15,
+        height: 25,
+        alignItems: 'center',
+        marginLeft: '2%',
+        marginBottom: '3%',
+        marginTop: '4%',
+        justifyContent: 'center'
+    },
+    iconSettings: {
+        marginLeft: '20%',
+        marginRight: '0%',
+    },
+    edtBtnTitleText: {
+        color: '#FFFFFF',
+        fontWeight: 'bold',
+        width: '66%',
+        textAlign: 'left',
+        marginLeft: '17%',
+        marginTop: '5%',
+        marginBottom: '3%',
+        fontSize: 19, 
+        alignItems: 'center'
+    },
+    edtBtnText: {
+        color: '#FFFFFF',
+        fontWeight: 'bold',
+        width: '45%',
+        textAlign: 'center'
+    },
```

Comment:
These look like they do the same thing, or it looks like you have some elements in here that you don't need. 

`color: '#FFFFFF', fontWeight: 'bold', fontSize: 19,` on line 167, 168, and 174 doesn't do anything

Reply:
@JCherry022 did you update this?

Reply:
yes